examples/react-navigation/android-fragment/android/app/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import groovy.lang.Closure
+
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("com.facebook.react")
+}
+
+android {
+    namespace = "com.example.bottomnavigationviews"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "com.example.bottomnavigationviews"
```

Comment:
I think that the app namespace should rather be something like: `com.example.bugsnag.reactnative.fragments`